modules/openid_auth/MIT-LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright 2018 
```

Comment:
I think you can delete this. All our .gov open source work is Public Domain, so would not be covered by this license.

Reply:
Removed it